react-components/src/index.js

```diff
@@ -0,0 +1,2 @@
+import './examples/CopyTextArea';
+import './examples/ToolDetails';
```

Comment:
This ended up becoming the dev-mode entry point (that populates the `index.html` file in `npm start` dev mode), and the `lib.js` file is the entry point for the DE.